spec/system/candidate_interface/candidate_needs_to_provide_two_new_referees_with_separate_additional_referees_flag_on_spec.rb

```diff
@@ -0,0 +1,251 @@
+require 'rails_helper'
+
+RSpec.describe 'Candidate needs to provide 2 new referees' do
+  include CandidateHelper
+
+  scenario "Candidate provides a new referee because 2 didn't respond" do
+    given_i_am_signed_in_as_a_candidate
+    and_i_have_submitted_my_application
+    and_both_of_my_referees_havent_responded_within_a_reasonable_timeframe
+    and_the_separate_additional_referees_flag_is_on
+
+    when_i_visit_the_application_dashboard
+    then_i_see_that_i_need_new_references
+
+    when_i_visit_the_start_page
```

Comment:
Could you click on the banner here instead of visiting the page directly?

Reply:
So this was my idea originally but David explained why we should do it this way. Effectively we are trying to mimic the behaviour as if the candidate proceeded by choosing yes from the new view.

They definitely could add another referee by clicking the banner... however, they would then see the normal add new referee page. To mimic the flow more accurately we want them to proceed from the add_another_referee page which should redirect them instead to the slightly different add a second referee page :)